app/models/manageiq/providers/hawkular/middleware_manager/live_metrics_capture.rb

```diff
@@ -38,6 +43,14 @@ def collect_live_metric(metric, start_time, end_time, interval)
       process_data(metric, metrics)
     end
 
+    def collect_stats_metric(metric, start_time, end_time, interval)
+      validate_metric(metric)
+      starts = start_time.to_i.in_milliseconds
+      ends = end_time.to_i.in_milliseconds
+      bucket_duration = "#{interval}s"
```

Comment:
I read it in a confusing way - duration is end-start, no? what is duration in context of interval here?